### PR TITLE
Neo4j migration part 2

### PR DESCRIPTION
**Pull request by Withington**
Reviews are requested if you have time. Otherwise, I'll merge this in on 5 November.

Changed all calls to neo4j to get uri and password from the .cfg file (local.cfg or phenopolis.cfg).

Added test data sufficient to get the ```/my_patients``` page working locally, using neo4j.

Added test_my_patients.py. Other pages affected by changes were tested manually via the web app.

### CODE REVIEW

File: views/individual.py [hunk 1]
```diff
@@ -198,7 +198,8 @@ def get_feature_venn(patient):
     RETURN t.termId, t.name, g.gene_id, g.gene_name
     """ % patient
     print(s)
-    data=requests.post('http://localhost:57474/db/data/cypher',auth=('neo4j','1'),json={'query':s}).json()['data']
+    uri='http://'+app.config['NEO4J_HOST']+':'+str(app.config['NEO4J_PORT'])+'/db/data/cypher'
+    data=requests.post(uri,auth=('neo4j',app.config['NEO4J_PWD']),json={'query':s}).json()['data']
```
Comment (IsmailM) [inline views/individual.py:202]: Is the plan to change to this to: `neo4j_session.run` at a later date? If so do we need `app.config['NEO4J_PORT']` at all?

File: tests/test_my_patients.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+from __future__ import print_function
```
Comment (IsmailM) [inline tests/test_my_patients.py]: Why are we importing this?

**Review by IsmailM — APPROVED** [review]
👍

File: tests/test_my_patients.py (hunk 2 above)
Comment (Withington) [reply tests/test_my_patients.py]: Removed it, thanks.

File: views/individual.py (hunk 1 above)
Comment (Withington) [reply views/individual.py:202]: Yes, that's the plan. This change for now makes it possible for people to run Phenopolis locally with minimal effort - Dr Fu from University of Alabama is getting set up to use it.

**Comment by Withington** [comment]
Thanks for the review Ismail.

**Pull request merged**